### ISSUE TITLE
As Pho, I don´t want to see things, that aren´t ready to use. 

### ISSUE THREAD

**Issue by evaklopp** [buzzn/scrum#21]

**Comment by dottorer** [comment]
@evaklopp Discussed with @scsirdx if we can move this front end only story also in a next sprint to load balance front end work for him. Please take a look at story and prioritize.

**Comment by evaklopp** [comment]
yes @dottorer and @scsirdx, I also thought of this option, I added it to the product backlog now

**Comment by dottorer** [comment]
@evaklopp I think this is complete, can you test and move to done?

**Comment by evaklopp** [comment]
Whooooo, done and closing this issue!

**Comment by dottorer** [comment]
For completeness: To invoke the devmode (and open up unfinished areas), use:

https://staging-admin.buzzn.io/localpools?devmode=true
